torch/nn/modules/module.py

```diff
@@ -864,6 +864,10 @@ def _slow_forward(self, *input, **kwargs):
         return result
 
     def _call_impl(self, *input, **kwargs):
+        forward_call = (self._slow_forward if torch._C._get_tracing_state() else self.forward)
```

Comment:
Could you add a comment here to explain what this function is doing?
In particular, write explicitly that if no hook is there, just skip the whole function. Or maybe actually rename this into `_call_with_hooks()` and make the parent function skip it if there are no hooks at all in this Module?